VideoLocker/src/main/java/org/edx/mobile/base/BaseSingleFragmentActivity.java

```diff
@@ -171,4 +166,10 @@ protected void hideMessageInSitu(){
         }
     }
 
+    private void showErrorDialog(String message) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setMessage(message)
+                .setPositiveButton(R.string.label_ok, null)
+                .create().show();
```

Comment:
Lots of scenarios. Here's a few:
- Pressing home while low on memory
- Switching to another app
- Opening an edX push notification (though, we don't have those yet...)
- User opens task-switcher and swipes dismisses the edX task
- Device enters Doze mode